### PR TITLE
cmake: CMake instructions missing -c flag

### PR DESCRIPTION
**Pull request by alimaredia**
The CMake instructions under "Running a test
cluster" were missing the ceph.conf vstart
generates for them.

Also restart and stop with init-ceph instructions
weren't functional.

Signed-off-by: Ali Maredia <amaredia@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,23 +148,21 @@ To run a functional test cluster,
...
+	./bin/rados -c ceph.conf -p rbd bench 30 write
+	./bin/rbd -c ceph.conf create foo --size 1000
 
 To shut down the test cluster,
 
 	./path/to/ceph/src/stop.sh
 
-To start or stop individual daemons, the sysvinit script should work:
-
-	./bin/init-ceph restart osd.0
```
Comment (alimaredia) [inline README.md]: @liewegas I tried giving this init-ceph a ceph.conf file but the osd still didn't come back up. The command is meant to be run after stop.sh right?

Comment (liewegas) [inline README.md]: usually i do restart, or stop (in leiu of stop.sh)

Comment (alimaredia) [inline README.md]: Neither of those commands worked when I had the everything up. That doesn't sound good to me... @oritwas @tchaikov @jcsp do any of you use init-ceph frequently with CMake + vstart.sh?

Comment (tchaikov) [inline README.md]: i don't use `init-ceph` along with CMake + vstart.sh.

File: README.md [hunk 2]
```diff
@@ -148,23 +148,21 @@ To run a functional test cluster,
 
 	cd build
 	./path/to/ceph/src/vstart.sh -d -n -x -l
-	./bin/ceph -s
+	./bin/ceph -c ceph.conf -s
```
Comment (tchaikov) [inline README.md]: why shall we specify the path to ceph.conf?

File: README.md [hunk 3]
```diff
@@ -148,23 +148,21 @@ To run a functional test cluster,
 
 	cd build
 	./path/to/ceph/src/vstart.sh -d -n -x -l
```
Comment (tchaikov) [inline README.md]: for the sake of consistency, could probably just use `../src/vstart.sh`? as we suggest
```
	cd build
	cmake [options] ..
	make check
```
before

Comment (liewegas) [inline README.md]: +1

File: README.md (hunk 2 above)
Comment (alimaredia) [inline README.md]: I think it defaults to searching in /etc/ceph for the ceph.conf. When I ran just the ceph command (without specifying the ceph.conf in $PWD) it hung.

**Comment by liewegas** [comment]
-c isn't needed for me.  and let's fix init-ceph.

**Pull request closed without merging**